src/mats_l1_processing/photometer.py

```diff
@@ -0,0 +1,93 @@
+#%%
+import numpy as np
+from mats_l1_processing.instrument import Photometer
+from pandas import DataFrame
+
+def calibrate_pm(df: DataFrame, photometer: Photometer):
+    
+# ===============================================================================================================
```

Comment:
Personally I find that it makes the code less readable, when comments ar not on the same level as the code it refers to, but I guess that is a matter of preference. Not sure what PEP says.